Expecta/NSArray+Difference.m

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2020 Lightricks. All rights reserved.
+// Created by Maxim Grabarnik.
+
+#import "NSArray+Difference.h"
+
+#import <assert.h>
+
+#import "ExpectaSupport.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+API_AVAILABLE(ios(13.0))
+typedef NSOrderedCollectionChange<NSObject *> Change;
+
+API_AVAILABLE(ios(13.0))
+@interface ChangePair : NSObject
+
+- (instancetype)init NS_UNAVAILABLE;
+
+- (instancetype)initWithInsertion:(Change *)insertion removal:(Change *)removal;
+
+@property (nonatomic, readonly) Change *insertion;
+
+@property (nonatomic, readonly) Change *removal;
+
+@end
+
+@implementation ChangePair
+
+- (instancetype)initWithInsertion:(Change *)insertion removal:(Change *)removal {
+  if (self = [super init]) {
+    _insertion = insertion;
+    _removal = removal;
+  }
+  return self;
+}
+
+@end
+
+@implementation NSArray (Difference)
+
+- (nonnull NSArray<NSString *> *)differenceFrom:(id)object
```

Comment:
IMO you can remove the `nonnull` since everything is wrapped inside `NS_ASSUME_NONNULL_BEGIN`